platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/fan.py

```diff
@@ -209,6 +209,18 @@ def set_status_led(self, color):
         status = False
         return status
 
+    def get_status_led(self):
+        """
+        Gets the state of the Fan status LED
+
+        Returns:
+            A string, one of the predefined STATUS_LED_COLOR_* strings.
+        """
+        if self.get_status():
+            return self.STATUS_LED_COLOR_GREEN
+        else:
+            return self.STATUS_LED_COLOR_OFF
+
 	def get_target_speed(self):
```

Comment:
The indentation is wrong here. It will break the reboot cause determination other cli commands which are related to sonic_platform.

Reply:
this is actually fixed in https://github.com/Azure/sonic-buildimage/pull/3363
Which one do you want to get in first?

Reply:
3363 is merged so you got the conflicts on this PR. please resolve the conflicts.

Reply:
Latest Merge conflicts have been resolved.